src/_includes/content/boiler.md

```diff
@@ -2,7 +2,7 @@
 
 B01LER is a fully customisable design system (DS) that enables you to easily create cohesive and consistent digital products across multiple platforms and brands.
 
-B01LER provides pre-built components that exist **as designs** in the form of a [component library in Figma](https://boilerds.com/B01LER%20%20%5Bv0.1%20-%20alpha%20release%5D.fig), and **as code** in the form of a [forkable repository](https://github.com/deven-org/B01LER-Kitchen).
+B01LER provides pre-built components that exist **as designs** in the form of a [component library in Figma](https://boilerds.com/B01LER%20%20%5Bv0.1%20-%20alpha%20release%5D.fig), and **as code** in the form of a [forkable repository](https://github.com/deven-org/boiler).
```

Comment:
The Figma link also was updated: https://www.figma.com/file/kG44mSWhEp2VcMvMqbJrRY/B01LER-%5Bv0.2--alpha-release%5D?type=design&node-id=1134%3A44297&mode=design&t=D8whh5YvFTH1immB-1